### ISSUE TITLE
data source: org-table

### ISSUE THREAD

**Issue by hajovonta** [larsmagne/eplot#9]
I'm not sure how big this request is, but I have data in the format of an org-table:

|       date |   value |
|------------+------|
| 2020-07-04 | 2054 |
| 2020-10-30 | 2064 |
| 2020-11-02 | 2065 |
| 2020-11-11 | 2111 |
| 2020-11-20 | 2215 |
| 2020-11-23 | 2258 |
| 2020-11-27 | 2333 |
| 2020-11-30 | 2390 |
| 2020-12-05 | 2483 |
| 2020-12-08 | 2541 |
| 2020-12-12 | 2606 |
| 2020-12-13 | 2617 |
| 2020-12-15 | 2647 |

Is it possible to use it as a data source? I can convert it to a plt if needed easily, but it would make plotting from org easier if we could just call eplot on a table.

**Comment by hajovonta** [comment]
To add more to this, it would be nice if it would plot all columns on the chart, but somehow it should be possible to indicate the x-axis data format (date series in this example) and perhaps the column that will be the x-axis.

**Comment by larsmagne** [comment]
This is basically the same problem as when plotting CSV files (which I added earlier today), so adding
this format should be simple enough.

**Comment by larsmagne** [comment]
(If you have a multi-plot Org test file, that would be convenient.)

**Comment by hajovonta** [comment]
Here is one example:

|       date | value1 | value2 | value3 |
|------------+--------+--------+--------|
| 2021-05-26 |     10 |     12 |     59 |
| 2021-05-27 |     12 |     18 |     60 |
| 2021-05-29 |     15 |     25 |     61 |
| 2021-05-30 |     18 |     32 |     49 |
| 2021-05-31 |     20 |     41 |     45 |
| 2021-06-01 |     20 |     36 |     46 |
| 2021-06-02 |     21 |     35 |     47 |
| 2021-06-03 |     23 |     38 |     50 |

**Comment by larsmagne** [comment]
Thanks; I'll have a look at the problem, but probably not until Wednesday.

**Comment by hajovonta** [comment]
I've noticed the date format is YYYYMMDD, is it possible to use also YYYY-MM-DD (ISO8601 accepts both)?

**Comment by hajovonta** [comment]
Have you had a chance to look at it?

**Comment by larsmagne** [comment]
Yeah, I had a peek, and it looked trivial to implement.  But I haven't had time to actually do so yet.  Perhaps this week.

**Comment by larsmagne** [comment]
I've now implemented this -- but I'm not really familiar with Org, so it's not unlikely that it'll break with other Org tables.  But `M-x eplot' will now use the Org parsing functions in org-mode buffers and try to make sense of the data.  But there's probably better ways to do this, so if it doesn't work in general, patches are welcome.
